# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/numpy/setup-rollback-23.1.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- Numeric-23.3/setup.py	2004-06-07 23:52:01.000000000 +0200
+++ Numeric-23.1/setup.py	2003-04-14 20:01:17.000000000 +0200
@@ -33,21 +33,20 @@
 
 # delete all but the first one in this list if using your own LAPACK/BLAS
 sourcelist = [os.path.join('Src', 'lapack_litemodule.c'),
-#              os.path.join('Src', 'blas_lite.c'), 
-#              os.path.join('Src', 'f2c_lite.c'), 
-#              os.path.join('Src', 'zlapack_lite.c'),
-#              os.path.join('Src', 'dlapack_lite.c')
+              os.path.join('Src', 'blas_lite.c'), 
+              os.path.join('Src', 'f2c_lite.c'), 
+              os.path.join('Src', 'zlapack_lite.c'),
+              os.path.join('Src', 'dlapack_lite.c')
              ]
 # set these to use your own BLAS;
 
-library_dirs_list = ['/usr/lib/atlas']
-libraries_list = ['lapack', 'cblas', 'f77blas', 'atlas', 'g2c'] 
-                   # if you also set `use_dotblas` (see below), you'll need:
-                   # ['lapack', 'cblas', 'f77blas', 'atlas', 'g2c']
+library_dirs_list = []
+libraries_list = [] # if you also set `use_dotblas` (see below), you'll need:
+                    # ['lapack', 'cblas', 'f77blas', 'atlas', 'g2c']
 
 # set to true (1), if you also want BLAS optimized matrixmultiply/dot/innerproduct
-use_dotblas = 1
-include_dirs = ['/usr/include/atlas']  # You may need to set this to find cblas.h
+use_dotblas = 0
+include_dirs = []  # You may need to set this to find cblas.h
                    #  e.g. on UNIX using ATLAS this should be ['/usr/include/atlas'] 
 
 # The packages are split in this way to allow future optional inclusion
